src/components/AppLayout.tsx

```diff
@@ -39,14 +43,14 @@ export default function AppLayout({ children }: React.PropsWithChildren) {
         <Disclosure as="nav" className="bg-gray-800">
           {({ open }) => (
             <>
-              <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
-                <div className="flex h-16 items-center justify-between">
+              <div className="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8">
+                <div className="flex items-center justify-between h-16">
```

Comment:
There are a lot of unrelated changes in this PR that make it really really hard to review. :)

Reply:
On it